components/builder-api/src/http/handlers.rs

```diff
@@ -17,6 +17,7 @@
 use bodyparser;
 use depot::server::check_origin_access;
 use hab_core::package::Plan;
+use hab_core::event::*;
```

Comment:
(minor style nit) 

Rust imports tend to be very explicit about the bits you are pulling in and avoiding wildcard imports.  It's not just a style thing, but can help prevent future breakage due to overlapping names (You can only have one of a non-module qualified name defined at any point.)

Source - https://aturon.github.io/style/imports.html#avoid-use-*,-except-in-tests.